controller/instance_manager_controller.go

```diff
@@ -498,6 +499,27 @@ func (imc *InstanceManagerController) syncInstanceStatus(im *longhorn.InstanceMa
 	return nil
 }
 
+func (imc *InstanceManagerController) syncDateEngineCPUMask(im *longhorn.InstanceManager) (bool, error) {
```

Comment:
If I understand well the purpose of this functions, it doesn't actually _sync_ masks, it only make some checks. If this is right, maybe the name of the function is a bit confusing.

Reply:
Renamed it to `isDateEngineCPUMaskApplied`. Thank you.